code/game/turfs/floor.dm

```diff
@@ -184,89 +174,90 @@ var/list/wood_icons = list("wood", "wood-broken")
 
 /turf/open/floor/is_plasteel_floor()
 	if(istype(floor_tile,/obj/item/stack/tile/plasteel))
-		return 1
+		return TRUE
 	else
-		return 0
+		return FALSE
 
 /turf/open/floor/is_light_floor()
 	if(istype(floor_tile,/obj/item/stack/tile/light))
-		return 1
+		return TRUE
 	else
-		return 0
+		return FALSE
 
 /turf/open/floor/is_grass_floor()
 	if(istype(floor_tile,/obj/item/stack/tile/grass))
-		return 1
+		return TRUE
 	else
-		return 0
+		return FALSE
 
 /turf/open/floor/is_wood_floor()
 	if(istype(floor_tile,/obj/item/stack/tile/wood))
-		return 1
+		return TRUE
 	else
-		return 0
+		return FALSE
 
 /turf/open/floor/is_carpet_floor()
 	if(istype(floor_tile,/obj/item/stack/tile/carpet))
-		return 1
+		return TRUE
 	else
-		return 0
+		return FALSE
 
 /turf/open/floor/is_plating()
 	if(!floor_tile)
-		return 1
-	return 0
+		return TRUE
+	return FALSE
 
 /turf/open/floor/proc/break_tile()
-	if(!breakable_tile || hull_floor) return
-	if(broken) return
+	if(!breakable_tile || hull_floor)
+		return
+	if(broken)
+		return
 	broken = TRUE
 	if(is_plasteel_floor())
 		icon_state = "damaged[pick(1, 2, 3, 4, 5)]"
-		broken = 1
-	else if(is_light_floor())
+
+	if(is_light_floor())
```

Comment:
There are no early returns here and several of the options are mutually-exclusive. A `switch` would also work.